README.md

```diff
@@ -62,6 +62,52 @@ Contacts.getContact( contactIdentifier, (error, contact) =>  {
 });
 ```
 
+#### Delete a Single Contact
+
+```js
+let contactIdentifier = 'A7806266-6574-4731-82E1-C54946F63E1C';
+
+Contacts.deleteContact( contactIndetifier, (error, contactId) => {
+  if (error) {
+    console.log(error);
+  }
+  else {
+    console.log(contactId);
+  }
+}
+```
+
+#### Create a Single Contact
+
+```js
+let data = {
+  'givenName': 'John',
+  'familyName': 'Appleseed',
+  'organizationName': 'Apple Inc',
+  'phoneNumbers': [
+    // (NOTE)
+    // Available labels 'home', 'work', 'mobile'
+    {'label': 'home', 'number': '555-522-8243'},
+    {'label': 'work': 'number': '(408) 555-5270'},
+  ],
+  emailAddresses: [
+    // (NOTE)
+    // Available labels 'work', 'home'
+    {'label': 'work', 'email': 'john.appleseed@apple.com'},
```

Comment:
Same as my previous comment, see if you export these label values as Constants instead: 

https://facebook.github.io/react-native/docs/native-modules-ios.html#exporting-constants